BedrockServer.h

```diff
@@ -206,10 +206,18 @@ class BedrockServer : public SQLiteServer {
     // When a peer node logs in, we'll send it our crash command list.
     void onNodeLogin(SQLiteNode::Peer* peer);
 
-    // Control the command port. The server will toggle this as necessary, unless manualOverride is set,
-    // in which case the `suppress` setting will be forced.
+    // You must block and unblock the command port with *identical strings*.
+    void blockCommandPort(const string& reason);
```

Comment:
Wouldn't `closeCommandPort` and `openCommandPort` be better names? Not sure...

Reply:
I don't think so, because they don't necessarily do that, which is why I chose `block`. If you send two `close` commands, only the first one actually closes it, and if you then send two `open` commands, only the second one actually opens it.

Reply:
Could we do `addCommandPortBlock` and `removeCommandPortBlock`?

Reply:
Is that clearer? I guess we could, but I'm not really sold it's any better.

Reply:
Maybe it's a bit clearer. What do you think @bondydaa @rafecolton ?

Reply:
No strong opinion one way or the other.

Reply:
meh